content/tutorials/single-page-tutorials/create-nft-with-wallet-lib.md

```diff
@@ -0,0 +1,197 @@
+# Create a NFT with wallet library and IPFS
+
+In this tutorial you will upload your desired image to IPFS and create a NFT from the unique IPFS URL on the Shimmer testnet. This tutorial will build on the tutorial "Setup testnet address and receive funds from faucet" ([IOTA Wiki PR under review](https://github.com/iota-community/iota-wiki/pull/711)). If you haven't completed it, please do so and afterwards come back here. You will use the same folder and development environment as in the mentioned prerequired tutorial, which will allow you to reuse the Stronghold account with funds on it.
+
+## Prerequisites
+
+- Installed [Node.js](https://nodejs.org/en/)
+- Finished [Setup testnet address and receive funds from faucet](https://wiki.iota.org/wallet-setup) tutorial
+
+## Setup
+
+### Prepare development environment
+
+Navigate to the folder of the prerequired tutorial [Setup testnet address and receive funds from faucet](https://wiki.iota.org/wallet-setup):
+
+
+```bash
+cd wallet-setup
+```
+
+Next to the existing dependencies of the wallet setup tutorial, you will only need to add two more packages:
+
+```bash
+npm install @iota/client
+npm install ipfs-core@0.10.8
+```
+
+Afterwards your `package.json` should at least contain the following dependencies:
+
+```json

```

Comment:
```suggestion
```json {6-12}
```

Reply:
this way we highlight the relevant lines :)

Reply:
Great. I was already looking on how to show this feature. Didn't think about this part ❤️